### ISSUE TITLE
iosTest fails with Null pointer exception loading bundle

### ISSUE THREAD

**Issue by dalewking** [icerockdev/moko-resources#98]
Maybe related to #48 #80 or #97, but translations are working when running the app, but when I try to run iosTest task I get a null pointer exception in this code:

```
actual object MR {
  private val bundle: NSBundle by lazy {
      NSBundle.loadableBundle("com.genesys.purecloud.wfmshared.MR") }
```

Presumably because the resource bundle cannot be found.

I cannot make any headway on trying to track this down

**Comment by RomeuG** [comment]
Is this in the sample code?

The sample code works for me. But if I try the project that I put the link for in #97 , I get a KotlinNullPointerException.
This is the stack trace:

```
Instances of kotlin.Error, kotlin.RuntimeException and subclasses aren't propagated from Kotlin to Objective-C/Swift.
Other exceptions can be propagated as NSError if method has or inherits @Throws annotation.
Uncaught Kotlin exception: kotlin.NullPointerException
        at 0   ResourceTest                        0x0000000100f67300 kfun:kotlin.Throwable.<init>()kotlin.Throwable + 72
        at 1   ResourceTest                        0x0000000100f60f9c kfun:kotlin.Exception.<init>()kotlin.Exception + 60
        at 2   ResourceTest                        0x0000000100f60c0c kfun:kotlin.RuntimeException.<init>()kotlin.RuntimeException + 60
        at 3   ResourceTest                        0x0000000100f61498 kfun:kotlin.NullPointerException.<init>()kotlin.NullPointerException + 60
        at 4   ResourceTest                        0x0000000100f98b28 ThrowNullPointerException + 116
        at 5   ResourceTest                        0x0000000101022e24 kfun:dev.icerock.moko.resources.utils.loadableBundle@platform.Foundation.NSBundle.Companion.(kotlin.String)platform.Foundation.NSBundle + 4040
        at 6   ResourceTest                        0x0000000101025b58 kfun:com.example.myapplication.sdk.MR.<init>$lambda-0#internal + 144
        at 7   ResourceTest                        0x0000000101025c88 kfun:com.example.myapplication.sdk.MR.$<init>$lambda-0$FUNCTION_REFERENCE$1.invoke#internal + 116
        at 8   ResourceTest                        0x0000000100f8a4f8 kfun:kotlin.native.concurrent.FreezeAwareLazyImpl.getOrInit#internal + 744
        at 9   ResourceTest                        0x0000000100f8ab6c kfun:kotlin.native.concurrent.FreezeAwareLazyImpl.<get-value>()T + 284
        at 10  ResourceTest                        0x000000010102337c kfun:com.example.myapplication.sdk.MR.<get-bundle>#internal + 272
        at 11  ResourceTest                        0x0000000101023598 kfun:com.example.myapplication.sdk.MR.strings.<init>()com.example.myapplication.sdk.MR.strings + 416
        at 12  ResourceTest                        0x0000000101051540 InitSharedInstanceStrict + 280
        at 13  ResourceTest                        0x0000000101026994 kfun:com.example.myapplication.sdk.Testing.getStrings()kotlin.collections.List<dev.icerock.moko.resources.desc.StringDesc> + 2536
        at 14  ResourceTest                        0x00000001010346d4 objc2kotlin.256 + 132
        at 15  ResourceTest                        0x0000000100f3b3ec $s12ResourceTest0aB3AppV7SwiftUI0C0AadEPxycfCTW + 12
        at 16  SwiftUI                             0x00000001c0dd22c4 $s7SwiftUI3AppPAAE4mainyyFZ + 124
        at 17  ResourceTest                        0x0000000100f3b360 $s12ResourceTest0aB3AppV5$mainyyFZ + 60
        at 18  ResourceTest                        0x0000000100f3b400 main + 12
        at 19  libdyld.dylib                       0x00000001813f3e60 20D5095F-60CD-3C1A-96AC-9A0583DC73BF + 3680
```

It seems it is on the `loadableBundle` part...

**Comment by dalewking** [comment]
It is on our project not sample code but that is the same error I get

**Comment by RomeuG** [comment]
So it is really directly related to #97 .

Have you tried the sample project?
I have, and it works there.

But I can't really grasp the reason why.

**Comment by Alex009** [comment]
problem not fixed in 0.11.1. reproducer here https://github.com/icerockdev/moko-resources/issues/99#issuecomment-684470108
will try fix in 0.12.0 (kotlin 1.3.72 version)

**Comment by Alex009** [comment]
@dalewking in 0.12.0 release implemented logic for ios tests with resources. sample `mpp-hierarhical` contains sample configuration of tests
